boot/storage.txt

```diff
@@ -0,0 +1,485 @@
+    work in including these filesystems in KISS! Others are available in 
+    user-created repositories. ZFS is one such example [8]. Due to licensing 
+    restrictions, ZFS requires additional work to use than other filesystems.
+
+    EXT4 is a solid, general purpose filesystem. For UEFI systems, FAT32 is a 
+    required filesystem for the BOOT partition. BTRFS is an experimental 
+    filesystem which sees heavily development by organizations like Facebook,
+    and offers a different way of managing storage than more traditional (EXT,
+    DOS, NTFS) filesystems. All of these have built-in kernel support.
+
+    For a more complete list of other filesystem options and what limitations 
+    and features exist for them, refer to the Arch Wiki [9].
+
+
+    [3.2] Swap Partition
+    ____________________________________________________________________________ 
+    
+    To set up a partition as a Linux swap space, the mkswap command is use. 
+    Replace X with the drive where the swap partition is located, and Y by the
+    partition on that drive that will be formatted as swap.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   mkswap /dev/sdXY                                                       |
+    |   swapon /dev/sdXY                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+  
+
+
+    [3.3] Swap file
+    ____________________________________________________________________________    
+    As an alternative to creating an entire partition, a swap file offers 
+    similar functionality, and the added benefits of being able to vary its size 
+    on-the-fly, as well as being more easily removed. This may be especially 
+    desirable if disk space is at a premium.
+
+    The most straightforward way to create a swap file is to use dd:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ dd if=/dev/zero of=$LOCATION bs=$BYTES count=$SIZE                   |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    $LOCATION is the partition where the swapfile should be located. For the 
+    next few commands, assume $LOCATION=/swapfile.
+
+    $BYTES is The block size, which is determined by the filesystem on the 
+    partition where $LOCATION is. This setting is important for optimizing
+    read/write throughput. dd defaults to 512 bytes, but this is suboptimal for 
+    drives with larger supported block sizes. To determine the supported block
+    size for your device, run dumpe2fs (provided by e2fsprogs):
```

Comment:
iirc there's some `/sys` or `/proc` file containing this information. I could be wrong though. I'll look into it.

Reply:
There should be; there's a way to make e.g. `busybox` print this information too. I'll find it. 

I had never actually heard of dumpe2fs so I was very excited to include it.